validation_tests/hpctoolkit/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+. ../../setup.sh
+
+SPACK_ROOT=`spack location -r`
+. ${SPACK_ROOT}/share/spack/setup-env.sh
+
+spack install hpctoolkit@develop
```

Comment:
The testsuite assumes the spack packages are already installed and spack is in the environment. if you do spackLoadUnique <package spec> (provided by the root setup.sh) it will load the package, do some other background work and set up some environment variables useful to some tests (though it doesn't look like HPCToolkit needs them).